Lombiq.Hosting.Tenants.FeaturesGuard/Handlers/FeaturesEventHandler.cs

```diff
@@ -108,7 +108,7 @@ private static Task HandleConditionallyEnabledFeaturesAsync()
     /// <returns>A boolean value whether ConditionallyEnabledFeaturesOptions is populated or not.
     /// Also produces <paramref name="featuresToEnable"/> and <paramref name="featuresToDisable"/>.
     /// </returns>
-    private static bool GetFeaturesToBeEnabledAndDisabled(
+    private static bool TryGetFeaturesToBeEnabledAndDisabled(
```

Comment:
Methods returning `bool` and having `out` parameters follow this convention in .NET.